Ryujinx.HLE/HOS/Services/Hid/IHidServer.cs

```diff
@@ -604,11 +615,12 @@ public ResultCode DeactivateNpad(ServiceCtx context)
         // AcquireNpadStyleSetUpdateEventHandle(nn::applet::AppletResourceUserId, uint, ulong) -> nn::sf::NativeHandle
         public ResultCode AcquireNpadStyleSetUpdateEventHandle(ServiceCtx context)
         {
+            PlayerIndex npadId        = HidUtils.GetIndexFromNpadIdType((NpadIdType)context.RequestData.ReadInt32());
             long appletResourceUserId = context.RequestData.ReadInt64();
-            int  npadId               = context.RequestData.ReadInt32();
             long npadStyleSet         = context.RequestData.ReadInt64();
 
-            if (context.Process.HandleTable.GenerateHandle(_npadStyleSetUpdateEvent.ReadableEvent, out int handle) != KernelResult.Success)
+            KEvent e = context.Device.Hid.Npads.GetStyleSetUpdateEvent(npadId);
```

Comment:
```suggestion
            KEvent evnt = context.Device.Hid.Npads.GetStyleSetUpdateEvent(npadId);
```
(Since event is a C# name)